src/lib/helpers/config.ts

```diff
@@ -17,31 +19,71 @@ export const ConfigSchema = z.object({
         ),
     }).strict().default({}),
     cache: z.object({
-        enabled: z.boolean().default(true),
-        directory: z.string().default("/var/tmp"),
+        enabled: z.boolean().default(
+            Deno.env.get("CACHE_ENABLED") === "true"
+                ? true
+                : Deno.env.get("CACHE_ENALBED") === "false"
+                ? false
+                : true,
+        ),
```

Comment:
same as with `JOBS_YOUTUBE_SESSION_PO_TOKEN_ENABLED` - my feeling is this is clearer as:
```suggestion
        enabled: z.boolean().default(
            Deno.env.get("CACHE_ENALBED") === "false"
                ? false
                : true,
        ),
```

Reply:
Tnanks!